ansys_optical_automation/application/example_windshield_distortion_lpf_reader.py

```diff
@@ -156,16 +180,33 @@ def plot_result(data_result):
     """
     x = []
     y = []
-    value = []
+    distortion_value = []
+    diopter_value = []
+    distortion_max = 0
+    diopter_max = 0
     for item in data_result:
-        x.append(item.start_position[0])
-        y.append(item.start_position[1])
-        if item.optical_power > 0.1:
-            value.append(0)
-        else:
-            value.append(item.optical_power)
-    plt.scatter(x, y, c=value, s=20)
-    plt.colorbar()
+        x.append(item.start_position[1])
```

Comment:
why no build s Speos xmp?

Reply:
added a function to write result into XMP file